concourse/client.py

```diff
@@ -134,8 +134,12 @@ def resource_check_webhook(
         resource_name: str,
         webhook_token: str,
         concourse_id: str,
+        job_mapping_name: str=None,
     ):
-        query_args = urlencode({'webhook_token': webhook_token, HOOK_CONCOURSE_ID: concourse_id})
+        query = {'webhook_token': webhook_token, HOOK_CONCOURSE_ID: concourse_id}
+        if job_mapping_name is not None:
```

Comment:
no need to introduce this in a backwards-compatible way - do the full required change in one commit, please